backend/btrixcloud/crawlconfigs.py

```diff
@@ -108,6 +109,9 @@ def __init__(
         self.coll_ops = cast(CollectionOps, None)
 
         self.default_filename_template = os.environ["DEFAULT_CRAWL_FILENAME_TEMPLATE"]
+        self.default_crawler_image_pull_policy = os.environ[
+            "DEFAULT_CRAWLER_IMAGE_PULL_POLICY"
+        ]
```

Comment:
```suggestion
        self.default_crawler_image_pull_policy = os.environ.get(
            "DEFAULT_CRAWLER_IMAGE_PULL_POLICY", "IfNotPresent"
        )
```

might not be a bad idea to have an extra backup here just in case?